pkg/app/testdata/app_template_test/show-only

```diff
@@ -0,0 +1,113 @@
+processing file "helmfile.yaml" in directory "."
+changing working directory to "/path/to"
+first-pass rendering starting for "helmfile.yaml.part.0": inherited=&{default  map[] map[]}, overrode=<nil>
+first-pass uses: &{default  map[] map[]}
+first-pass rendering output of "helmfile.yaml.part.0":
+ 0: 
+ 1: releases:
+ 2: - name: logging
+ 3:   chart: incubator/raw
+ 4:   namespace: kube-system
+ 5: 
+ 6: - name: kubernetes-external-secrets
+ 7:   chart: incubator/raw
+ 8:   namespace: kube-system
+ 9:   needs:
+10:   - kube-system/logging
+11: 
+12: - name: external-secrets
+13:   chart: incubator/raw
+14:   namespace: default
+15:   labels:
+16:     app: test
+17:   needs:
+18:   - kube-system/kubernetes-external-secrets
+19: 
+20: - name: my-release
+21:   chart: incubator/raw
+22:   namespace: default
+23:   labels:
+24:     app: test
+25:   needs:
+26:   - default/external-secrets
+27: 
+28: 
+29: # Disabled releases are treated as missing
+30: - name: disabled
+31:   chart: incubator/raw
+32:   namespace: kube-system
+33:   installed: false
+34: 
+35: - name: test2
+36:   chart: incubator/raw
+37:   needs:
+38:   - kube-system/disabled
+39: 
+40: - name: test3
+41:   chart: incubator/raw
+42:   needs:
+43:   - test2
+44: 
+
+first-pass produced: &{default  map[] map[]}
+first-pass rendering result of "helmfile.yaml.part.0": {default  map[] map[]}
+vals:
+map[]
+defaultVals:[]
+second-pass rendering result of "helmfile.yaml.part.0":
+ 0: 
+ 1: releases:
+ 2: - name: logging
+ 3:   chart: incubator/raw
+ 4:   namespace: kube-system
+ 5: 
+ 6: - name: kubernetes-external-secrets
+ 7:   chart: incubator/raw
+ 8:   namespace: kube-system
+ 9:   needs:
+10:   - kube-system/logging
+11: 
+12: - name: external-secrets
+13:   chart: incubator/raw
+14:   namespace: default
+15:   labels:
+16:     app: test
+17:   needs:
+18:   - kube-system/kubernetes-external-secrets
+19: 
+20: - name: my-release
+21:   chart: incubator/raw
+22:   namespace: default
+23:   labels:
+24:     app: test
+25:   needs:
+26:   - default/external-secrets
+27: 
+28: 
+29: # Disabled releases are treated as missing
+30: - name: disabled
+31:   chart: incubator/raw
+32:   namespace: kube-system
+33:   installed: false
+34: 
+35: - name: test2
+36:   chart: incubator/raw
+37:   needs:
+38:   - kube-system/disabled
+39: 
+40: - name: test3
+41:   chart: incubator/raw
+42:   needs:
+43:   - test2
+44: 
+
+merged environment: &{default  map[] map[]}
+WARNING: release test2 needs disabled, but disabled is not installed due to installed: false. Either mark disabled as installed or remove disabled from test2's needs
+1 release(s) matching name=logging found in helmfile.yaml
+
+processing 1 groups of releases in this order:
+GROUP RELEASES
+1     default/kube-system/logging
+
+processing releases in group 1/1: default/kube-system/logging
+changing working directory back to "/path/to"
```

Comment:
why need this?

Reply:
Do you mean, why did I need to add this file?
It was created when I ran `go test -v $PKGS -coverprofile cover.out -race -p=1` (after setting `PKGS=$(go list ./... | grep -v /vendor/ | tr '\n' ' ')`)

The first time running the tests without the file the test failing:
```
    --- FAIL: TestTemplate/show-only (0.02s)
```
And the `Expected` is empty when the file doesn't exist.
```
        	Error:      	Not equal: 
        	            	expected: ""
        	            	actual  : "processing file \"helmfile.yaml\" in directory \".\"\nchanging working directory to \"/path/to\"\nfirst-pass rendering starting for \"helmfile.yaml.part.0\": inherited=&{default  map[] map[]}, overrode=<nil>\nfirst-pass uses: &{default  map[] map[]}\nfirst-pass rendering output of \"helmfile.yaml.part.0\":\n 0: \n 1: releases:\n 2: - name: logging\n 3:   chart: incubator/raw\n 4:   namespace: kube-system\n 5: \n 6: - name: kubernetes-external-secrets\n 7:   chart: incubator/raw\n 8:   namespace: kube-system\n 9:   needs:\n10:   - kube-system/logging\n11: \n12: - name: external-secrets\n13:   chart: incubator/raw\n14:   namespace: default\n15:   labels:\n16:     app: test\n17:   needs:\n18:   - kube-system/kubernetes-external-secrets\n19: \n20: - name: my-release\n21:   chart: incubator/raw\n22:   namespace: default\n23:   labels:\n24:     app: test\n25:   needs:\n26:   - default/external-secrets\n27: \n28: \n29: # Disabled releases are treated as missing\n30: - name: disabled\n31:   chart: incubator/raw\n32:   namespace: kube-system\n33:   installed: false\n34: \n35: - name: test2\n36:   chart: incubator/raw\n37:   needs:\n38:   - kube-system/disabled\n39: \n40: - name: test3\n41:   chart: incubator/raw\n42:   needs:\n43:   - test2\n44: \n\nfirst-pass produced: &{default  map[] map[]}\nfirst-pass rendering result of \"helmfile.yaml.part.0\": {default  map[] map[]}\nvals:\nmap[]\ndefaultVals:[]\nsecond-pass rendering result of \"helmfile.yaml.part.0\":\n 0: \n 1: releases:\n 2: - name: logging\n 3:   chart: incubator/raw\n 4:   namespace: kube-system\n 5: \n 6: - name: kubernetes-external-secrets\n 7:   chart: incubator/raw\n 8:   namespace: kube-system\n 9:   needs:\n10:   - kube-system/logging\n11: \n12: - name: external-secrets\n13:   chart: incubator/raw\n14:   namespace: default\n15:   labels:\n16:     app: test\n17:   needs:\n18:   - kube-system/kubernetes-external-secrets\n19: \n20: - name: my-release\n21:   chart: incubator/raw\n22:   namespace: default\n23:   labels:\n24:     app: test\n25:   needs:\n26:   - default/external-secrets\n27: \n28: \n29: # Disabled releases are treated as missing\n30: - name: disabled\n31:   chart: incubator/raw\n32:   namespace: kube-system\n33:   installed: false\n34: \n35: - name: test2\n36:   chart: incubator/raw\n37:   needs:\n38:   - kube-system/disabled\n39: \n40: - name: test3\n41:   chart: incubator/raw\n42:   needs:\n43:   - test2\n44: \n\nmerged environment: &{default  map[] map[]}\nWARNING: release test2 needs disabled, but disabled is not installed due to installed: false. Either mark disabled as installed or remove disabled from test2's needs\n1 release(s) matching name=logging found in helmfile.yaml\n\nprocessing 1 groups of releases in this order:\nGROUP RELEASES\n1     default/kube-system/logging\n\nprocessing releases in group 1/1: default/kube-system/logging\nchanging working directory back to \"/path/to\"\n"
```

Running the test again with this file works.